lib/main.coffee

```diff
@@ -2,6 +2,16 @@
 helpers = require 'atom-linter'
 path = require 'path'
 
+isGreaterVersion = (v1, v2) ->
+  v1Parts = v1.split('.')
+  v2Parts = v2.split('.')
+  minLength = Math.min(v1Parts.length, v2Parts.length)
+  if minLength > 0
+    for idx in [0..minLength - 1]
+      unless Number(v1Parts[idx]) == Number(v2Parts[idx])
+        return Number(v1Parts[idx]) < Number(v2Parts[idx])
+  return v1Parts.length < v2Parts.length
```

Comment:
The problem to eliminate is `0.10.0.dev.99999` is a lower version than `0.10.0`.

`=` and `<=` are both ok.

We always use semver for real versions, so `v1Parts.length = 3` would also work.

Ohhh , but ... `0.11.0.dev.123` needs to be higher than `0.10.0`.

Doesnt coffee have a decent library for semver?? ;-)